.github/workflows/ccpp-tests.yml

```diff
@@ -0,0 +1,56 @@
+name: C/C++ CI Unit Tests
+
+on:
+  push:
+    branches: [master]
+    paths-ignore:
+      - 'python/**'
+      - 'tools/extra/fpgabist/**'
+      - 'tools/extra/packager/*.py'
+      - 'tools/extra/packager/metadata/**'
+      - 'tools/extra/packager/test/*.py'
+      - 'tools/extra/pac_hssi_config/*.py'
+      - 'tools/extra/fpgadiag/**'
+      - 'tools/utilities/**'
+      - 'scripts/*.py'
+      - 'platforms/scripts/platmgr/**'
+      - '.github/workflows/python-static-analysis.yml'
+  pull_request:
+    branches: [master]
+    paths-ignore:
+      - 'python/**'
+      - 'tools/extra/fpgabist/**'
+      - 'tools/extra/packager/*.py'
+      - 'tools/extra/packager/metadata/**'
+      - 'tools/extra/packager/test/*.py'
+      - 'tools/extra/pac_hssi_config/*.py'
+      - 'tools/extra/fpgadiag/**'
+      - 'tools/utilities/**'
+      - 'scripts/*.py'
+      - 'platforms/scripts/platmgr/**'
+      - '.github/workflows/python-static-analysis.yml'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        test-script: [cover.sh, unit-tests.sh]
+    steps:
+    - uses: actions/checkout@v2
+    - name: update
+      run: sudo apt-get update -y
+    - name: Get Packages
+      uses: mstksg/get-package@v1
+      with:
+        apt-get: uuid-dev libjson-c-dev libhwloc-dev lcov libtbb-dev
+    - name: set hugepages
+      run: sudo sysctl -w vm.nr_hugepages=8
+    - name: run ${{ matrix.test-script }}
+      run: ${{ github.workspace }}/scripts/${{ matrix.test-script }}
+    - name: Coveralls
+      if: ${{ matrix.test-script == 'cover.sh' }}
+      uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.github_token }}
```

Comment:
@r-rojo I don't see the coveralls reports appearing in the CI.

Perhaps `secrets.github_token` is case sensitive?
https://github.com/marketplace/actions/coveralls-github-action